src/components/Chat/Chat.tsx

```diff
@@ -269,12 +272,16 @@ export const Chat = memo(() => {
         confirm(t<string>('Are you sure you want to clear all messages?')) &&
         conversation
       ) {
-        const { messages } = conversation;
+        const { messages, isNameChanged, name } = conversation;
+        const newConversationName = isNameChanged
+          ? name
+          : DEFAULT_CONVERSATION_NAME;
 
         dispatch(
           ConversationsActions.updateConversation({
             id: conversation.id,
             values: {
+              name: newConversationName,
```

Comment:
You should do this not here but where you have changing name for empty chats, because you are not covered case when user will delete messages one by one

Reply:
Thank you for pointint at this case.
Moved condition to the sendMessageEpic